internal/task/main.go

```diff
@@ -0,0 +1,88 @@
+package task
+
+import (
+	"fmt"
+	"strconv"
+	"sync"
+	"time"
+)
+type Task struct{
+	Task_type string
+	Name string
+	Min_time int
+	Max_time int
+
+}
+type Tasks struct { 
+    Secondly []*Task
+    Minutely []*Task
+    Hourly    []*Task
+}
+func NewTask (a []string) *Task{
+	if len(a) != 5{
+			fmt.Println("Ошибка! Неверная запись в кофигурационном файле, ожидается 5 аргументов через пробел, получено", len(a))
+			return nil
+	}
+	mi,err := strconv.Atoi(a[3])
+	if err != nil {
+		fmt.Println("Ошибка! Неверные аргументы функции, 2 аргумент должен быть числом")
+		return nil
+	}
+	ma,err := strconv.Atoi(a[4])
+	if err != nil {
+		fmt.Println("Ошибка! Неверные аргументы функции, 3 аргумент должен быть числом")
+		return nil
+	}
+	if mi>ma{
+		fmt.Println("Ошибка! Неверные аргументы функции, минимальное время выполнения функции превосходит максимальное")
+		return nil
+	}
+	task := Task{Task_type: a[0],Name: a[2], Min_time: mi, Max_time: ma }
```

Comment:
```suggestion
	return &Task{
	    Task_type: a[0],
	    Name: a[2], 
	    Min_time: mi, 
	    Max_time: ma,
     }
```
и нужно все названия привести в camelCase